persistence/src/androidMain/kotlin/com/wire/kalium/persistence/util/SecurityUtil.kt

```diff
@@ -0,0 +1,60 @@
+package com.wire.kalium.persistence.util
+
+import android.security.keystore.KeyGenParameterSpec
+import android.security.keystore.KeyProperties.BLOCK_MODE_GCM
+import android.security.keystore.KeyProperties.ENCRYPTION_PADDING_NONE
+import android.security.keystore.KeyProperties.KEY_ALGORITHM_AES
+import android.security.keystore.KeyProperties.PURPOSE_DECRYPT
+import android.security.keystore.KeyProperties.PURPOSE_ENCRYPT
+import java.security.KeyStore
+import javax.crypto.Cipher
+import javax.crypto.KeyGenerator
+import javax.crypto.SecretKey
+import javax.crypto.spec.GCMParameterSpec
+
+class SecurityUtil {
```

Comment:
Shouldn't these security methods be responsibility of the `cryptography`module?

Reply:
not really this is not related to `Proteus`, `SecurityUtil` is simply and encryption layer on top off `DataStore` (dataStore files are not encripted by default) and moving this to `cryptography` will add an un unnecessary complexity